Sources/PointFree/Account/UpdateProfile.swift

```diff
@@ -58,13 +62,6 @@ private func updateProfileMiddlewareHandler(
   let updateFlash: Middleware<HeadersOpen, HeadersOpen, Prelude.Unit, Prelude.Unit>
   if data.email.rawValue.lowercased() != user.email.rawValue.lowercased() {
     updateFlash = flash(.warning, "We've sent an email to \(user.email) to confirm this change.")
-    Task {
-      _ = try await sendEmail(
-        to: [user.email],
-        subject: "Email change confirmation",
-        content: inj2(confirmEmailChangeEmailView((user, data.email, emailChangePayload)))
-      )
-    }
```

Comment:
I moved this down to the async IO below, _and_ made it so that it's sent only if the `Current.database.updateUser` succeeds, which is probably more correct.